src/main/java/Duke.java

```diff
@@ -6,5 +8,40 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("------------------------------------------------");
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?\n");
+        System.out.println("------------------------------------------------");
+
+        Scanner in = new Scanner(System.in);
+        String input = in.nextLine();
+        String[] list = new String[100];
+        int listCounter = 0;
+        while(!input.equals("bye")){
+            if(input.equals("list")){
+                System.out.println("------------------------------------------------");
+                System.out.println("list");
+                for(int i=0; i<listCounter;i++){
+                    System.out.println(i+1 + ". " +  list[i]);
+                }
+                System.out.println("------------------------------------------------");
+            } else{
+                list[listCounter] = input;
+                listCounter++;
+                System.out.println("------------------------------------------------");
+                System.out.println("added: " + input);
+                System.out.println("------------------------------------------------");
+
+            }
+            in = new Scanner(System.in);
+            input = in.nextLine();
+
+        }
+        if(input.equals("bye")) {
```

Comment:
This if-check seems redundant, as once the while loop is broken, this condition will always be satisfied.

Reply:
I just noticed that. Thank you!